Minecraft/src/main/java/com/microsoft/Malmo/Client/ClientStateMachine.java

```diff
@@ -956,8 +958,10 @@ protected void execute() throws Exception
      */
     public class WaitingForServerEpisode extends ConfigAwareStateEpisode
     {
+        final int WAIT_MAX_TICKS = 1000;
```

Comment:
Why 1000 ticks?

Reply:
About 30 sec in server ticks but I will be be doubling to 1 min.